backend/nodejs/src/services/MonitoringService.ts

```
@@ -0,0 +1,132 @@
+        if (device.powerSettings.wakeCronEnabled && device.powerSettings.wakeCron) {
+          this.setupWakeCron(device);
+        }
+      });
+
+      logger.info(`Initialized monitoring for ${devices.length} devices`);
+    } catch (error) {
+      logger.error('Failed to initialize monitoring:', error);
+    }
+  }
+
+  public async startMonitoring(device: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)